lib/src/main/kotlin/WeatherSensorService/Public/WeatherSensorReadingType.kt

```diff
@@ -0,0 +1,22 @@
+//
+// SpinDance
+// Weather Sensor Service
+//
+// Copyright © 2022 SpinDance. All rights reserved.
+//
+
+import java.time.LocalDateTime
+
+public interface WeatherSensorReadingType {
+    /// Temperature in degrees Celcius
+    var temperature: Double
+
+    /// Percent humidity
+    var humidity: Double
+
+    /// Barometric pressure in kiloPascals
+    var pressure: Double
+
+    /// Reading timestamp
+    var time: LocalDateTime
```

Comment:
Seems like I might have commented on this elsewhere, but these should be immutable. Is there a restriction that interface properties have to be `var`? Seems like there isn't but I could be wrong.